pkg/tmpl/context_funcs.go

```diff
@@ -48,7 +49,8 @@ func (c *Context) createFuncMap() template.FuncMap {
 	return funcMap
 }
 
-func (c *Context) Exec(command string, args []interface{}, inputs ...string) (string, error) {
+// TODO: in the next major version, remove this function.
```

Comment:
Why is there this comment :/

Reply:
Yes. I think this function can be merged when the main version is changed, that is, exec and execEnv can be merged, which is a destructive modification.